src/snowflake/connector/auth/keypair.py

```diff
@@ -180,9 +180,6 @@ def update_body(self, body: dict[Any, Any]) -> None:
     def assertion_content(self) -> str:
         return self._jwt_token
 
-    def should_retry(self, count: int) -> bool:
-        return count < self._jwt_retry_attempts
-
```

Comment:
let's bring it back for now and create a JIRA for future BCR changes regarding the retry